src/Microsoft.ML.Data/Scorers/FeatureContributionCalculation.cs

```diff
@@ -0,0 +1,449 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Text;
+using Microsoft.ML.Data;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.CommandLine;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Internal.Internallearn;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using Microsoft.ML.Runtime.Model;
+using Microsoft.ML.Runtime.Numeric;
+
+[assembly: LoadableClass(typeof(IDataScorerTransform), typeof(BindableMapper), typeof(BindableMapper.Arguments),
+    typeof(SignatureDataScorer), "Feature Contribution Transform", "fct", "FeatureContributionCalculationTransform", MetadataUtils.Const.ScoreColumnKind.FeatureContribution)]
```

Comment:
>fct [](start = 68, length = 3)

I guess you didn't do this, but why is something that is intended to be used as a *scorer* being identified as a *transform*? That makes no sense. Certainly none of our other scorers do anything like this.

Reply:
I can fix that! Good point

---
In reply to: [243072199](https://github.com/dotnet/machinelearning/pull/1847#discussion_r243072199) [](ancestors = 243072199)